src/freenas/etc/ix.rc.d/ix-etc

```diff
@@ -5,12 +5,13 @@
 
 # PROVIDE: ix-etc
 # REQUIRE: earlykld
-# BEFORE: fsck
+# BEFORE: fsck smartd-daemon
```

Comment:
This really makes most sense to be put on smart-daemon

`REQUIRE: LOGIN ix-etc`